src/ducks/konnectors/index.js

```diff
@@ -6,6 +6,10 @@ export const DOCTYPE = 'io.cozy.konnectors'
 
 export const fetchKonnectors = () => fetchCollection('konnectors', DOCTYPE)
 
+// TODO: Fetch the registry
+export const fetchKonnectorInMaintenance = () =>
```

Comment:
Konnector should be pluralized here, am I right (`fetchKonnectorsInMaintenance`) ?